day3/slice_time_image.py

```diff
@@ -71,7 +97,13 @@ def slice_time_file(fname, slice_times, TR):
     TR: double
         Repetition time in seconds.
     """
-    # Hint: use os.path.split and os.path.join to make the new filename
+    img = nib.load(fname)
+    interp_img = slice_time_image(img, slice_times, TR)
+    
+    path, tail = os.path.split(fname)
+    new_tail = 'Interp_' + tail  # I decided to use a different new filename
+    # I thought it would provide more information than just adding 'a'
```

Comment:
Fair enough :)

SPM uses `a` for slice-time-corrected files, where I think `a` stands for "acquisition" time corrected.